flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/JobMasterTest.java

```diff
@@ -1044,6 +1050,7 @@ public void testResourceManagerConnectionAfterStart() throws Exception {
      * if this execution fails.
      */
     @Test
+    @Category(FailsWithAdaptiveScheduler.class) // FLINK-21399
```

Comment:
instead we could reference FLINK-21450; then I don't have to touch this test when I address FLINK-21399 🙈